tools/machine-learning/mujoco/interactive_viewer.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
I initially got the following wayland error: 
```sh
GLFWError: (65544) b'Wayland: Window position retrieval not suppor
ted'
  warnings.warn(message, GLFWError)
```
The viewer still opened, but was completely unresponsive to inputs.
After some searching for that error, I realized i hadn't installed `glfw` locally. Installing `glfw` via my distros pacakge manager, fixed the problem. Does that mean this is a global dependency for this to work, or can we perhaps find a workaround that solves this internally in the workspace?

Reply:
I think this is a global dependency, mujoco uses it for its renderer.

https://mujoco.readthedocs.io/en/3.1.1/programming/samples.html#sasimulate